zeek/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - zeek
+
```

Comment:
```suggestion
<!-- towncrier release notes start -->

```

Reply:
@iliakur Can you please help us understand, if this is some default that needs to be added?

Reply:
@iliakur could you conditionally add this to the template if the integration is in core so we do not have to add it manually?

Reply:
Yea, we have another thing that we need to conditionally do in templates. I'm guessing this is pretty urgent, since we have more integrations incoming.

Reply:
@bhargavnariyanicrest please apply my suggested change, otherwise lgtm